public/components/game/details/details.js

```diff
@@ -335,8 +335,13 @@ exports.ViewModel = CanMap.extend(
 	 * ```
 	 */
 	gotoTimeMinus5: function(time, event) {
-		this.attr("youtubePlayer").seekTo(time - 5, true);
-		this.attr("youtubePlayer").playVideo();
+		var player = this.attr("youtubePlayer");
+		if (player.seekTo) {
+			player.seekTo(time - 5, true);
+		}
+		if (player.playVideo) {
+			player.playVideo();
+		}
```

Comment:
That's right.

On Thursday, March 24, 2016, Justin Meyer <notifications@github.com> wrote:

> In public/components/game/details/details.js
> <https://github.com/donejs/bitballs/pull/242#discussion_r57408606>:
>
> > @@ -335,8 +335,13 @@ exports.ViewModel = CanMap.extend(
> >  	 * ```
> >  	 */
> >  	gotoTimeMinus5: function(time, event) {
> > -		this.attr("youtubePlayer").seekTo(time - 5, true);
> > -		this.attr("youtubePlayer").playVideo();
> > +		var player = this.attr("youtubePlayer");
> > +		if (player.seekTo) {
> > +			player.seekTo(time - 5, true);
> > +		}
> > +		if (player.playVideo) {
> > +			player.playVideo();
> > +		}
>
> player would exist, but not these methods?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/donejs/bitballs/pull/242/files/ed619f60a8cc0f0be9dc8a428c7e2ce2baab3425#r57408606>
>